proto/route.proto

```diff
@@ -43,7 +43,7 @@ message Route {
 
 // ENI route table with CA prefix for packet Outbound
 message RouteKey {
-    string eni = 1;
+    string group_id = 1;
```

Comment:
Will this change break swss Azp?

Reply:
oh yeah good point, will put together a PR to update orchagent/swss to reflect the name change. Will hold off on merging this PR until the SWSS PR is ready.

Reply:
deprecated ENI field and added group_id as a new field to avoid breaking change